From 13a0c56a642cc94c4c5cb95f7b96dc817dbd3859 Mon Sep 17 00:00:00 2001
From: Adam Borowski <kilobyte@angband.pl>
Date: Fri, 6 Apr 2018 01:11:38 +0200
Subject: [PATCH 3/3] Recommend arch-test.

Machines that run debootstrap in non-d-i scenarios do this for a chroot or
lxc or similar guest, thus can be assumed to have abundant storage, making
accurate error messages worth a small extra package.
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 612eaa5..ce0fc0b 100644
--- a/debian/control
+++ b/debian/control
@@ -12,7 +12,7 @@ Vcs-Git: https://anonscm.debian.org/git/d-i/debootstrap.git
 Package: debootstrap
 Architecture: all
 Depends: ${misc:Depends}, wget
-Recommends: gnupg, ${keyring}
+Recommends: gnupg, ${keyring}, arch-test (>= 0.11~)
 Suggests: ubuntu-archive-keyring
 Description: Bootstrap a basic Debian system
  debootstrap is used to create a Debian base system from scratch,
-- 
2.17.0

